### ISSUE TITLE
Upgrade webpack-preprocessor for babel-loader 9

### ISSUE THREAD

**Issue by dobladov** [cypress-io/cypress#24435]
### Current behavior

After the new release of `babel-loader` `9.0.0` the installation on a project with `webpack-preprocessor` produces a dependency error.

![Screenshot 2022-10-28 at 12 14 16](https://user-images.githubusercontent.com/1938043/198566548-f8b23e2d-99f6-4c7a-9d52-b6436e52f7aa.png)

https://github.com/cypress-io/cypress/blob/develop/npm/webpack-preprocessor/package.json#L40

### Desired behavior

The installation of the newer babel version should not cause package install issues.

### Test code to reproduce

Install `babel-loader` 9 in a project with the latest `@cypress/webpack-preprocessor`

### Cypress Version

10.11.0

### Node version

v19.0.0

### Operating System

mac Ventura 13

### Debug Logs

```shell
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @cypress/webpack-preprocessor@5.15.0
npm ERR! Found: babel-loader@9.0.0
npm ERR! node_modules/babel-loader
npm ERR!   dev babel-loader@"9.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer babel-loader@"^8.0.2" from @cypress/webpack-preprocessor@5.15.0
npm ERR! node_modules/@cypress/webpack-preprocessor
npm ERR!   dev @cypress/webpack-preprocessor@"5.15.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: babel-loader@8.2.5
npm ERR! node_modules/babel-loader
npm ERR!   peer babel-loader@"^8.0.2" from @cypress/webpack-preprocessor@5.15.0
npm ERR!   node_modules/@cypress/webpack-preprocessor
npm ERR!     dev @cypress/webpack-preprocessor@"5.15.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```
```


### Other

_No response_

**Comment by anna-mkrtchyan-vertex** [comment]
Hey! I'm having the same issue when trying to install project dependencies from GitHub workflow using cypress-io/github-action@v5.
It uses **npm ci** to install dependencies

Local installation worked fine using **npm install**

@cypress/webpack-preprocessor - 5.16.1
babel-loader version - 9.1.2
cypress version - 12.3.0
cypress-io/github-action version - 5
Node version - 16.16.0

Does this require separate fix from your side, or smth may be wrong with my configuration?

![image](https://user-images.githubusercontent.com/87694052/218758109-9b0578e2-bc14-42e2-8fea-010335a33473.png)

**Comment by anna-mkrtchyan-vertex** [comment]
btw, **npm install** from GitHub workflow didn't work as well

![image](https://user-images.githubusercontent.com/87694052/218990080-fa2465c2-64f2-4b6c-ad59-7c64ed214ac8.png)

**Comment by dobladov** [comment]
@anna-mkrtchyan-vertex in my case it was solved after updating to **@cypress/webpack-preprocessor** `5.16.3`, I'm unsure if `5.16.1` corrects the problem that you have, that version is more than two months old, I believe the problem was solved for `5.16.2` or `5.16.3`

**Comment by anna-mkrtchyan-vertex** [comment]
@dobladov thanks a lot! It solved my issue as well! :)
